src/pages/Inventory/ListItems.js

```diff
@@ -1,7 +1,100 @@
-import React from "react";
-
+import React, { Fragment, useEffect, useState, useCallback } from "react";
+import withWrapper from "common/withWrapper";
+import Navbar from "common/Navbar";
+import { FormattedMessage } from "react-intl";
+import { Link } from "react-router-dom";
+import { useNotification } from "notification";
 const ListItems = () => {
-    return <div>I am list items page</div>;
+    const [tableData, setTableData] = useState([]);
+    useEffect(() => {
+        fetchData();
+    }, []);
+    const { triggerNotification } = useNotification();
+
+    // Function to fetch data from local storage
+    const fetchData = useCallback(() => {
+        let inventoryData = [];
+        if (localStorage.getItem("inventory_data")) {
+            try {
+                inventoryData = JSON.parse(
+                    localStorage.getItem("inventory_data")
+                );
+            } catch (e) {
+                triggerNotification("Failed parsing inventory data", {
+                    type: "error",
+                });
+                localStorage.removeItem("inventory_data");
+            }
+        }
+        setTableData(inventoryData);
+    }, [tableData]);
+
+    return (
+        <Fragment>
+            <Navbar opened="inventory" />
+            <div className="page-content p-5 bg-primary">
+                <div className="page-heading-wrapper p-5 mb-5">
+                    <span className="title">
+                        {" "}
+                        <FormattedMessage id="title.items"></FormattedMessage>{" "}
```

Comment:
can't we use like this 
`<FormattedMessage id="title.items" />` ?

Reply:
@gautism1